src/app/cube/browse/browse.component.scss

```diff
@@ -36,7 +36,11 @@ $learning-object-width: 820px;
     max-width: calc((#{$max-width} - #{$learning-object-width}) / 2 - 50px);
     min-width: 210px;
     width: 100%;
+    height: 650px;
     margin-right: 50px;
+    background-color: white;
+    box-shadow: 0 1px 12px 2px rgba(0, 0, 0, 0.05);
```

Comment:
Can this box-shadow be the same value of the box-shadow used elsewhere on the page? It's too deep and isn't consistent with the rest of the page.

Reply:
This box shadow is now changed with the one you sent me